src/js/actions/transform.js

```diff
@@ -231,7 +235,40 @@ define(function (require, exports) {
         // Used to calculate the new top/left positions of all layers in relation to top/left
         // of the group
         var overallWidthRatio = size.w ? size.w / overallBounds.width : 0,
-            overallHeightRatio = size.h ? size.h / overallBounds.height : 0;
+            overallHeightRatio = size.h ? size.h / overallBounds.height : 0,
+            overallLeftOffset = 0,
+            overallTopOffset = 0,
+            direction;
+
+        if (size.hasOwnProperty("w")) {
```

Comment:
I am pretty sure the existing pattern is wrong and the inputs are optional so I will change it in both places.